examples/browser/package.json

```diff
@@ -19,17 +19,12 @@
     "@theia/console": "^0.14.0",
     "@theia/core": "^0.14.0",
     "@theia/debug": "^0.14.0",
-    "@theia/debug-nodejs": "^0.14.0",
```

Comment:
we should consume node-debug and node-debug2 to from somewhere, they are built-in but bundled from marketplace by VS Code

Reply:
We will likely need to fork and create our own release since they do not have an appropriate `.vsix` release. It is a prerequisite for `@theia/vscode-builtin-debug-auto-launch` #6630

Reply:
@marcdumais-work should we create forks under **theia-ide** for these two repositories so we can successfully package and release `.vsix` files. I assume this is temporary until we have our own registry.

Reply:
@vince-fugnitto We already have them: https://github.com/theia-ide/vscode-node-debug2/releases and https://github.com/theia-ide/vscode-node-debug/releases . 

I last built the `.vsix` last september, but it's not a big deal to built/publish a more recent version.

Reply:
@marcdumais-work thank you! I've added them to the list of plugins along with `debug-auto-launch`.